pymeasure/instruments/thorlabs/thorlabspm100usb.py

```diff
@@ -34,11 +34,11 @@
 class ThorlabsPM100USB(Instrument):
     """Represents Thorlabs PM100USB powermeter."""
 
-    def __init__(self, adapter, **kwargs):
+    def __init__(self, adapter, name="ThorlabsPM100USB powermeter", **kwargs):
         super().__init__(
-            adapter, "ThorlabsPM100USB powermeter", **kwargs
+            adapter, name, **kwargs
         )
-        self.timout = 3000
+        self.timout = 3000  # TODO what is this for?
```

Comment:
IDK, maybe @bleykauf remembers?

Reply:
It was added by @dvspirito in the original addition of this instrument.

At that time, this (misspelled) property was not used either, so it was always an unused attribute.

I push a commit to remove this line (it is not used anyways).

Reply:
@bilderbuchi feel free to commit my suggestion and merge.

Reply:
Sounds good.